.gitignore

```diff
@@ -29,6 +29,7 @@ share/python-wheels/
 MANIFEST
 metagpt/tools/schemas/
 examples/data/search_kb/*.json
+metagpt/ext/sela/AutogluonModels
```

Comment:
no need to ignore a particular folder. users will handle them themselves

Reply:
done